tlair/src/test/scala/at/forsyte/apalache/tla/lir/IrGenerators.scala

```diff
@@ -80,6 +80,12 @@ trait IrGenerators extends TlaType1Gen {
       TlaSetOper.in, TlaSetOper.notin, TlaSetOper.map, TlaSetOper.powerset, TlaSetOper.recSet, TlaSetOper.seqSet,
       TlaSetOper.setminus, TlaSetOper.subseteq, TlaSetOper.times, TlaSetOper.union)
 
+  /**
+   * Function operators (`TlaFunOper._`)
+   */
+  val functionOperators = List(TlaFunOper.enum, TlaFunOper.tuple, TlaFunOper.app, TlaFunOper.domain, TlaFunOper.funDef,
+      TlaFunOper.recFunDef, TlaFunOper.recFunRef, TlaFunOper.except)
```

Comment:
I think there's very limited value in supporting `recFunDef` and `recFunRef`. Not only are they the next to go, they will also generate countless malformed expressions, due to the restrictions on their proper use.

Reply:
I am not certain yet about letting the recursive functions to go (in contrast to recursive operators). I have seen some value in them recently. Though they can always be replaced with `CHOOSE` over a function set.

Reply:
Tested this on the Keramelizer PBT with `minSuccessful = 10,000` and it didn't produce a malformed expression, so I think it's fine in this regard. If `reckFun{Def,Ref}` become obsolete, we can clean them out then.